src/main/java/frc/robot/subsystems/climber/ClimberConstants.java

```diff
@@ -0,0 +1,48 @@
+package frc.robot.subsystems.climber;
+
+public class ClimberConstants {
+
+  private static final String CONSTRUCTOR_EXCEPTION = "constant class";
+
+  private ClimberConstants() {
+    throw new IllegalStateException(CONSTRUCTOR_EXCEPTION);
+  }
+
+  public static final boolean DEBUGGING = false;
+  public static final boolean TESTING = false;
+  public static final String CLIMBER = "Climber";
+
+  public static final int LEFT_MOTOR_CAN_ID = 15;
+  public static final int RIGHT_MOTOR_CAN_ID = 16;
+
+  // FIXME: update gear ratio
+  public static final double GEAR_RATIO = 25.0;
+  public static final boolean MOTOR_INVERTED = false;
+
+  public static final double LEFT_POSITION_PID_P = 0.0;
```

Comment:
Only one set of constants is needed for left and right. They will be the same. Usually we name them KP, KI, KD, KS, KV, and KA. PID refers to the P, I, and D constants. KS, KV, KA are not part of a PID, they are feedforward constants.